packages/styling/src/ScreenShareOverlay/ScreenShareOverlay-theme.scss

```diff
@@ -0,0 +1,19 @@
+.str-video__screen-share-overlay {
+  background: rgba(0, 0, 0, 0.9);
```

Comment:
Should we expose variables for this? Also, I think we should reduce the opacity to at least `0.8`. I think it is useful for presenters to be able to see what they are presenting :)

Reply:
I'll either reuse the overlay color we have or create another one alongside the one we have. As for the opacity - design specification does not have this at all, this was my addition but we can bump it down to .8. 👍

Reply:
![localhost_3000_join_lKie2H3pSTgi (1)](https://github.com/GetStream/stream-video-js/assets/43254280/1410701d-d466-4787-aa42-69a76460d6a8)

This is what it looks like with the overlay color we already have defined - what do you think?